README.md

```
@@ -1,42 +1,31 @@
-# <img src="assets/NSS-128x128.png" align="left" /> AppVeyor (and GitHub!) Artifacts Receiver
+# <img src="assets/NSS-128x128.png" align="left" /> AppVeyor Artifacts Receiver
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)